src/phpDocumentor/Pipeline/Stage/Configure.php

```diff
@@ -51,7 +52,7 @@ public function __construct(
     public function __invoke(array $options) : array
     {
         $this->configFactory->addMiddleware(
-            new CommandlineOptionsMiddleware($options, $this->configFactory, getcwd())
+            new CommandlineOptionsMiddleware($options, $this->configFactory, 'file:///' . getcwd())
```

Comment:
getcwd already returns an absolute path; shouldn't there thus be only 2 slashes?

Reply:
That depends on the os😭